nav/nav_stack/config/base_local_planner_params.yaml

```diff
@@ -0,0 +1,23 @@
+# Navigation Function (navigation path)
+NavfnROS:
+  # Specifies whether or not to allow navfn to create plans that traverse unknown space.
+  allow_unknown: true
+  # A tolerance on the goal point for the planner.
+  default_tolerance: 0.1
+
+# Plans robot movement
+TrajectoryPlannerROS:
+  max_vel_x: 2.0
+  min_vel_x: 0.2
+  max_vel_theta: 1.0
+  min_in_place_vel_theta: 0.4
```

Comment:
Are these values set according to the IGVC rules/requirements?

Reply:
We'll update these to reflect the following:
- Min speed = 1mph = 0.44704 m/s 
- Max speed = 5mph = 2.2352 m/s
(not sure about theta)